pkg/scene/builder/scene.go

```diff
@@ -62,11 +65,19 @@ func (b *Builder) widgets(ctx context.Context, s *scene.Scene, p []*property.Pro
 			PluginID:    w.Plugin().String(),
 			ExtensionID: string(w.Extension()),
 			Property:    b.property(ctx, findProperty(p, w.Property())),
+			Extended:    w.WidgetLayout().Extended,
 		})
 	}
 	return res
 }
 
+func (b *Builder) widgetAlignment(ctx context.Context, s *scene.Scene) *widgetAlignSystemJSON {
```

Comment:
```suggestion
func (b *Builder) widgetAlignSystem(ctx context.Context, s *scene.Scene) *widgetAlignSystemJSON {
```

The names should be consistent.